<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEncrypt</class>
 <widget class="QWidget" name="FileEncrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>73</y>
     <width>571</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="keyLabel">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyLineEdit">
        <property name="placeholderText">
         <string>Enter your key.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="encodeButton">
          <property name="text">
           <string>Encode from File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decodeButton">
          <property name="text">
           <string>Decode to File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="dataTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="placeholderText" stdset="0">
       <string>If you are decoding, paste your encoded data here.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
